kubernetes-examples/multi-tenant/README.md

```diff
@@ -0,0 +1,12 @@
+# Multi Tenant Example on Kubernetes.
```

Comment:
I did minimal work on the READMEs for the the other examples. I don't think we need to care about these too much right now.